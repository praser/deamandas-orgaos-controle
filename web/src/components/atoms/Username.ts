import styled from 'styled-components';

const Username = styled.p`
  /* Layout */
  display: block;
  margin-block: unset;

  /* Text */
  font-family: Nunito, -apple-system, system-ui, 'Segoe UI', Roboto,
    'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
    'Segoe UI Symbol', 'Noto Color Emoji';
  font-size: 12.8px;
  font-weight: 600;
  line-height: 19.2px;
  text-align: left;

  /* Appearance */
  color: rgb(133, 135, 150);
`;

export default Username;
