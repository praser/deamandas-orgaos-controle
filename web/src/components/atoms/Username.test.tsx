import React from 'react';
import { render, screen } from '@testing-library/react';
import Username from './Username';

describe('Username atomic component', () => {
  const name = 'User name';
  let username: HTMLElement;

  beforeEach(() => {
    render(<Username>{name}</Username>);
    username = screen.getByText(name);
  });

  it('is expected to match the snapshot', () => {
    expect(username).toMatchSnapshot();
  });
});
