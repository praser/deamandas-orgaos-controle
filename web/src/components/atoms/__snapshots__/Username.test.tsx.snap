// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Username atomic component is expected to match the snapshot 1`] = `
<p
  class="sc-bdvvaa wtevO"
>
  User name
</p>
`;
